src/SME.SGP.Dados/Repositorios/RepositorioPendencia.cs

```diff
@@ -96,5 +96,16 @@ public async Task AtualizarStatusPendenciasPorIds(long[] ids, SituacaoPendencia
             var query = @"update pendencia set situacao = @situacaoPendencia where id =ANY(@ids)";
             await database.Conexao.ExecuteAsync(query, new { ids, situacaoPendencia });
         }
+
+        public async Task<IEnumerable<PendenciaPendenteDto>> ObterPendenciasPendentes()
+        {
+            var query = @"select p.id as PendenciaId, p.ue_id as UeId
+                          from pendencia p
+                          where p.situacao = @situacao
+                   and excluido is false
+                   and p.ue_id is not null;";
+            var idsPendenciasPendentes = await database.Conexao.QueryAsync<PendenciaPendenteDto>(query, new { situacao = SituacaoPendencia.Pendente });
+            return idsPendenciasPendentes.AsList().ToArray();
```

Comment:
Não precisa usar o AsList().ToArrray()
```suggestion
            return await database.Conexao.QueryAsync<PendenciaPendenteDto>(query, new { situacao = SituacaoPendencia.Pendente });
```